_topic_map.yml

```diff
@@ -142,6 +142,8 @@ Topics:
     Distros: openshift-enterprise,openshift-origin
   - Name: Configuring your firewall
     File: configuring-firewall
+  - Name: Configuring cluster-wide proxy
```

Comment:
_Configuring cluster-wide proxy support_ or _Configuring the cluster-wide proxy_?

Reply:
+1 for _Configuring the cluster-wide proxy_